app/models/hud_reports/generator_base.rb

```diff
@@ -116,5 +116,23 @@ def self.question_fields(_question)
     def self.common_fields
       []
     end
+
+    def self.allowed_options
```

Comment:
Maybe `HudReport::BaseController` should use this list as well, so that the displayed and setable options match?

Reply:
Maybe, but I think I'd rather have the report allow additional options (at least for now.)   That way if we add some to the form, but forget to add them here, they'll correctly affect the calculation even if they don't show up.  Plus, I think the setting of `@filter` is delegated down to the report type, and these would need to be formatted differently to work in the params context.